src/Component/Pages/TodoListPage/TodoList.tsx

```diff
@@ -47,20 +74,31 @@ const TodoList: React.FC<ITodoList> = ({
                       backgroundColor: "lightblue",
                     }}
                   >
-                    <TableCell align="center">Id</TableCell>
+                    <TableCell align="center">Completed</TableCell>
+                    <TableCell align="center">Unique UUId</TableCell>
                     <TableCell align="center">Name</TableCell>
                     <TableCell align="center">Todo Task</TableCell>
                     <TableCell align="center">Actions</TableCell>
                   </TableRow>
                 </TableHead>
                 <TableBody>
                   {todoItems.map(
-                    (inputValue: {
-                      todo: string;
-                      id: number;
-                      userName: string;
-                    }) => (
+                    (
+                      inputValue: {
+                        todo: string;
+                        id: string;
+                        userName: string;
+                        completed: boolean;
+                      },
+                      index: number
+                    ) => (
                       <TableRow key={inputValue.id}>
```

Comment:
It's better to write a unique name, in this case : `TodoList_todo_${inputValue.id}`

Reply:
refactor: modified the todoitems.map key as this: key={`TodoList_todo_${todo.id}`}